pkgs/development/python-modules/notifications-android-tv/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, python3
+, fetchPypi
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "notifications-android-tv";
+  version = "1.0.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-jrKauDkgDyEJOLl25yeGgJDHOirDIPNgQi/z0wVNRD4=";
+  };
+
+  buildInputs = with python3.pkgs; [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
```

`with python3.pkgs;` is no longer needed with the change mentioned above

```suggestion
  propagatedBuildInputs = [
```

---

I don't see how that'd work as is? Where is `httpx` coming from? Right now, applying these suggestions leads to unknown variables for the (propagated) build inputs. Wouldn't poetry-core and httpx also need to be part of the derivation's parameters?

---

When you use python's callPackage function, which is what happens when you use python-packages.nix, you get your python dependecies via the argument list of your function.